lib/ui/mockData/MockUserProfileViewModel.dart

```diff
@@ -45,18 +48,18 @@ class MockUserProfileViewModel {
 }
 
 class MockUserProfileListItemViewModel {
-  static UserProfileListItemViewModel build(index) {
-    return UserProfileListItemViewModel(
-      title: _mockActionTitle[index],
+  static ProfileListItemViewModel build(index) {
+    return ProfileListItemViewModel(
+      title: Intl.message(_mockActionTitle[index]),
```

Comment:
That should be removed, it's in mock area, probably you got it from rebase.

Reply:
done